src/forms/SelectInput.jsx

```diff
@@ -31,75 +31,23 @@ export class SelectInput extends React.PureComponent {
 	}
 
 	render() {
-		const {
-			children,
-			className,
-			labelClassName,
-			label,
-			options,
-			id,
-			name,
-			error,
-			errors,
-			onChange, // eslint-disable-line no-unused-vars
-			value, // eslint-disable-line no-unused-vars
-			helperText,
-			required,
-			requiredText,
-			...other
-		} = this.props;
-
-		const classNames = {
-			label: cx(
-				'label--field',
-				{
-					'label--required': required,
-					'flush--bottom': helperText,
-				},
-				labelClassName
-			),
-			field: cx(
-				'select--reset span--100 padding--selectArrow',
-				{ 'field--error': (errors && errors.length > 0) || error },
-				className
-			),
-			helperText: cx('helperTextContainer', { required }),
-		};
+		// eslint-disable-next-line no-unused-vars
+		const { options, error, errors, requiredText, ...other } = this.props;
```

Comment:
Looks like you're no longer using requiredText but this was something that was added for the schedule form, we ended up deprioritizing adding it, but the ticket I created for this still exists and I believe it's something Sylvan and the org success team want to eventually tackle.
Here's the ticket: https://meetup.atlassian.net/browse/MG-441
So in order to complete this ticket, this component will need to support the requiredText prop.

Reply:
Update: I'm not sure the product requirement directly applies to the SelectInput/Select component but if we're changing required text from '*' to the word 'required' spelled out then it should be changed everywhere in mup-web which is what that ticket is for.

I don't necessarily agree with the product change, but this flexibility was here for a reason.

Reply:
Thanks for the context. I'll have to think about it a bit more, but yeah it looks like I'll have to add it back in. I _really_ think this is something that a design system should enforce consistency on rather than making it overridable, but given the interactions with TRNs, I don't see a clean and simple way of changing the copy to `(required)` universally... *le sigh*

Reply:
Yea, we only made it "override-able" because we don't want to handle translation in our tools

Reply:
Good call Jackie! I do agree though that we should be consistent. Defect is marked as important :)

Reply:
I've deviated slightly from the original `requiredText` implementation and instead made `required: boolean | string` to allow people to pass in custom 'required' text through that property. The reason for doing it that way is that it avoids having props that are dependent on each other (i.e. `requiredText` assumes that `required` is `true`) - if you supply a string to `required`, then the input will be required.

The fallback to `'*'` is still included, but no more need for a `defaultProp`